### PR TITLE
ci: skip nginx connectivity test with qemu/crio

### PR DESCRIPTION
**Pull request by littlejawa**
We have an error with service name resolution with this test when using qemu/crio. This error could not be reproduced outside of the CI for now. Skipping it to keep the CI job running until we find a solution.

See: https://github.com/kata-containers/kata-containers/issues/10414

### CODE REVIEW

File: tests/integration/kubernetes/k8s-nginx-connectivity.bats
Comment (fidencio) [reply tests/integration/kubernetes/k8s-nginx-connectivity.bats:12]: One way to do that would be exposing the `matrix.container_runtime` from https://github.com/kata-containers/kata-containers/blob/main/.github/workflows/run-k8s-tests-on-amd64.yaml#L44 to the `gha-run.sh` script, all the way to the tests ... and skip it if `container_runtime` is `crio`.

WDYT?

Comment (littlejawa) [reply tests/integration/kubernetes/k8s-nginx-connectivity.bats:12]: Yeah, I suspected it :-(
I've tried to look at all other skipped test and didn't find something for crio.

Is it OK if I add an environment variable to expose the `matrix.container_runtime` parameter?
Otherwise I feel my only option is to check the "KUBERNETES_EXTRA_PARAMS", but it could be misleadig/changing in subtle ways in the future.

Comment (littlejawa) [reply tests/integration/kubernetes/k8s-nginx-connectivity.bats:12]: ah, crossed comments, github didn't update fast enough :-p I'll do that

**Comment by fidencio** [comment]
@littlejawa, almost there.  Let's split this in 2 parts, otherwise we cannot properly test it without it getting merged as it is.

The first commit will be just about exposing the CONTAINER_RUNTIME in the actions, and in the gha-run.sh.  That can come as it's own PR that we'll merge it before.  Then this PR here can stay only with the skip, and it can actually be properly tested after the first one gets merged and this one gets rebased.

Does it sound reasonable?

**Comment by littlejawa** [comment]
> Does it sound reasonable?

Sure :)

**Comment by littlejawa** [comment]
Depends on: https://github.com/kata-containers/kata-containers/pull/10496

**Comment by gkurz** [comment]
> Depends on: #10496

Now merged. Please rebase.

**Review by fidencio — APPROVED** [review]
lgtm, assuming green tests.
Thanks @littlejawa!

**Review by gkurz — APPROVED** [review]
> lgtm, assuming green tests. Thanks @littlejawa!

FWIW some other issue could prevent green tests but we'd still want this to be merged.

Thanks @littlejawa !

**Pull request merged**
